app/templates/suppliers/_frameworks_open.html

```diff
@@ -1,41 +1,31 @@
- <div class='grid-row'> 
```

Comment:
I think we still probably want these in a grid row and column?  Even if it's a `column-one-whole`?  Or are these partials already inside a grid row element when they get included?